### DISCUSSION TITLE
Enterprise member invitations are disabled in this environment

### DISCUSSION THREAD

**Discussion by testpraveen** [community/community#142498] [Enterprise]
### Select Topic Area

Question

### Body

Trying to query _pendingUnaffiliatedMemberInvitations_ or running _inviteEnterpriseMember_ added in https://docs.github.com/en/graphql/overview/changelog#schema-changes-for-2024-06-13-1, and I am getting an error "Enterprise member invitations are disabled in this environment". How do I enable this?

**Comment by queenofcorgis** [comment]
Hey there @testpraveen - thank you so much for your patience. Are you in an EMU environment? Technically in an EMU you don't invite users to an Enterprise and there are no pending invites to the Enterprise- users are provisioned with SCIM.
